src/modules/story/story.service.ts

```diff
@@ -20,4 +20,18 @@ export class StoryService {
   async getStoriesForProject(projectId: number) {
     return await this.storyRepo.find({ where: { projectId }, order: { id: 'ASC' } });
   }
+
+  async findById(id: number) {
+    return await this.storyRepo.findOne(id);
+  }
+
+  async updateStory(data: VStoryOpt) {
```

Comment:
Tle je potencialno problem, da lahko kdorkoli editira katerkol story (tudi če nima pravic do njega), vse kar mora vedeti je id storija (ki ga lahko ugane, ker so integerji od 1 naprej). Nevem kolk se bo Fujs trudil iskat ranljivosti tho...

Js bi dodal zraven v argument userja, ki je naredil request (iz @AuthUser) in preveril da je v developerjih ali je master/owner.